tools/merge-docker-images.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+# Shorten the first argument (commit sha) to 7 chars
+SHORT_SHA=${1:0:7}
+TAG="sha-$SHORT_SHA"
+
+# Check if images for both amd64 and arm64 exist
+if docker pull clockworklabs/spacetimedb:$TAG-amd64 --platform amd64 >/dev/null 2>&1 && docker pull clockworklabs/spacetimedb:$TAG-arm64 --platform arm64 >/dev/null 2>&1; then
```

Comment:
This seems like it could be race-condition-ey if the images are built in parallel (you could have this script executed twice with both images available). Is this not the case or is it not a problem?

Reply:
It could potentially happen that it's executed twice, but it's not a problem - the one that will run later will just overwrite the existing manifest. This situation is unlikely, though, the window is probably like a second after the image is uploaded before the script runs.